### PR TITLE
Updated install instructions

### PR DESCRIPTION
**Pull request by guigsc**

### CODE REVIEW

**Review by thismat — COMMENTED** [review]
I appreciate the update. I'm not really a fan of the error highlighting that happens when using comments in JSON. I had that before as well and did not like the look and feel of it. 

I'm open to suggestions on changes. Maybe using it without syntax highlighting is the way to go?

File: INSTALL.md [hunk 1]
```diff
@@ -53,3 +53,24 @@ Example:
     }
 }
 ```
+
+If the profiles are listed as below:
+
+```json
```
Comment (NatoBoram) [inline INSTALL.md]:
```suggestion
```jsonc
```

File: INSTALL.md [hunk 2]
```diff
@@ -53,3 +53,24 @@ Example:
...
+"profiles": [
+    // list of profiles
+]
+```
+
+Change it to:
+
+```json
```
Comment (NatoBoram) [inline INSTALL.md]:
```suggestion
```jsonc
```

**Review by NatoBoram — CHANGES_REQUESTED** [review]
This should remove the errors.

**Comment by thismat** [comment]
@NatoBoram Very nice! Thanks!

**Pull request merged**